### DISCUSSION TITLE
Reset to Defaults

### DISCUSSION THREAD

**Discussion by geoffmcc** [rxhanson/Rectangle#903] [Q&A] [answered]
Just started using the app and it's pretty good. I am just wondering if there is a configuration file somewhere that gets left behind on uninstall. I installed using homebrew and then uninstalled to check out rectangle-pro, which I think I will upgrade too next week. I noticed when I reinstalled rectangle some of the settings I changed remain there, which leads me to believe there are some config files being left behind. If so could you point me in the direction of them please?

**Comment by rxhanson** [comment, marked as answer]
Thanks for checking out Rectangle & Pro!

Here is the information on the prefs storage https://github.com/rxhanson/Rectangle#preferences-storage.

Nearly all apps will use NSUserDefaults, and you can use the `defaults` terminal command to read/manipulate them. For example, you can delete the defaults using `defaults delete com.knollsoft.Rectangle`.

**Reply by rxhanson** [reply]
(There's also the reset shortcuts button in the settings tab of the prefs window, but that won't catch all the settings)
